packages/python/chart-studio/test_requirements/requirements_37.txt

```diff
@@ -6,6 +6,6 @@ retrying==1.3.3
 pytest==3.5.1
 pandas==0.23.2
 numpy==1.14.3
-ipywidgets==7.2.0
+anywidget
```

Comment:
Should we have a version pin here?

Reply:
Min version I think should be 0.9.0. If you want to pin an exact version, I'd use the latest: v0.9.13.

Reply:
Pinned!